api/PerformanceMeasure.json

```diff
@@ -0,0 +1,55 @@
+{
+  "api": {
+    "PerformanceMeasure": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PerformanceMeasure",
+        "support": {
+          "webview_android": {
+            "version_added": true
```

Comment:
I'm seeing Android support as `46`.

Reply:
Is the Android Browser the same as Android Webview? That was the reason I marked it `true`.